File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--creating-an-external-list-in-sharepoint-by-using-business-data_1.vb
    Public Shared Function ReadItem(ByVal contactID As Integer) As Contact
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim Contact As Contact = _
            (From TempContacts In dataContext.Contacts.AsEnumerable().Take(20) _
            Where TempContacts.ContactID = contactID _
            Select TempContacts).[Single]()
        Return Contact
    End Function

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--creating-an-external-list-in-sharepoint-by-using-business-data_1.cs
        public static Contact ReadItem(int contactID)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");

            Contact Contact =
                (from contacts in dataContext.Contacts.AsEnumerable().Take(20)
                 where contacts.ContactID == contactID
                 select contacts).Single();
            return Contact;
        }